src/index.html

```diff
@@ -22,6 +26,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__box">
+        <a
+          href="images/logo.png"
+          class="header__box-logo"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav class="header__container">
+        <div class="header__container-titles">
+          <a
+            class="is-active"
+            data-qa="card"
+          >
+            APPLE
+          </a>
+          <div class="menu">SAMSUNG</div>
+          <div class="menu">SMARTPHONES</div>
+          <div
+            data-qa="nav-hover"
+            class="menu"
+          >
+            LAPTOPS & COMPUTERS
+          </div>
+          <div class="menu">GADGETS</div>
+          <div class="menu">TABLETS</div>
+          <div class="menu">PHOTO</div>
+          <div class="menu">VIDEO</div>
+        </div>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__img"
+          src="././images/imac.jpeg"
+          alt="imac"
+        />
+        <div class="card__title">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__code">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__star">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p>Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-1">Price:</p>
+          <p class="card__price-2">$2,199</p>
+        </div>
+
+        <div class="card__bottom">
```

Comment:
just to reflect the correct property name, it should be:

`card__button` and `card__button-link`

Botao = Button